src/app/pages/Languages.svelte

```diff
@@ -61,9 +62,8 @@
   }
 
   .languages__container {
-    margin-left: 75px;
```

Comment:
Nitpick: What are your thoughts on adding more margin to the left?

I attached a comparison of the two desktop versions
https://imgur.com/a/aQmeVwd